src/app.ts

```diff
@@ -1147,7 +1168,31 @@ export class SlackApp<E extends SlackEdgeAppEnv | SlackSocketModeAppEnv> {
             return toCompleteResponse(slackResponse);
           }
         }
+      } else if (body.type === PayloadType.AppRateLimited) {
+        // App rate limited
+        const slackRequest: SlackRequest<E, AppRateLimited> = {
+          payload: body as AppRateLimited,
+          ...baseRequest,
+        };
+        // Only a single appRateLimited handler is supported
+        if (this.#appRateLimited) {
+          const handler = this.#appRateLimited(payload);
+          if (handler) {
+            if (!this.startLazyListenerAfterAck) {
+              ctx.waitUntil(handler.lazy(slackRequest));
+            }
+            const slackResponse = await handler.ack(slackRequest);
```

Comment:
Do you see the necessity of having an ack handler for this event? If all you need to (or can) do is just to return 200 OK like other standard Events API patterns, supporting only lazy listeners would be fine like [#events does](https://github.com/slack-edge/slack-edge/blob/1.3.8/src/app.ts#L972-L984).